test/source_map_utils_test.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {expect} from 'chai';
+
+import * as sm from '../src/source_map_utils';
```

Comment:
same comment here about module name, but the short name is okay-er here because it's a short file.